data/json/npcs/TALK_FACTION_CAMP.json

```diff
@@ -152,7 +169,7 @@
   {
     "id": "TALK_CAMP_TUTORIAL_EVAC_SHELTER",
     "type": "talk_topic",
-    "dynamic_line": "You can build a faction camp in the common evac shelter - the basic design with a single, wide open room.  Evac shelters are everywhere and often have fields surrounding them, so they're a little more flexible than a modular field camp but still make it easy to add expansions.\n  Evac center camps can be built up in several ways.  You basically need to decide between building more bedrooms or adding for crafting and cooking space.",
+    "dynamic_line": "You can build a faction camp in any kind of evac shelter, as long as it wasn't destroyed before you got there.  Evac shelters are everywhere and often have fields surrounding them, so they're a little more flexible than a modular field camp but still make it easy to add expansions.\n  Evac center camps can be built up in several ways.  You basically need to decide between building more bedrooms or adding for crafting and cooking space.",
```

Comment:
You basically need to decide between building more bedrooms or adding for crafting and cooking space >

The shelter will become the bunkhouse with a basic crafting area.  Expansions will allow for extended crafting areas.  The fire station camp offers most faction camp functions within a single map square to offset the lack of expansion space.